pkg/transports/connecting/dtls/client.go

```diff
@@ -34,9 +34,19 @@ type ClientTransport struct {
 	// // state tracks fields internal to the registrar that survive for the lifetime
 	// // of the transport session without being shared - i.e. local derived keys.
 	// state any
-	privPort int
-	pubPort  int
-	psk      []byte
+	privPort            int
+	pubPort             int
+	psk                 []byte
+	stunServer          string
+	disableIRWorkaround bool
+}
+
+type ClientConfig struct {
+	// STUNServer is the address of the stun server to use
+	STUNServer string
+
+	// DisableIRWorkaround disables sending an empty packet to workaround DTLS blocking in IR
```

Comment:
If we are going to have a name and a comment referencing DTLS blocking in IR could you add a brief description of what they are doing and when we have seen it? i.e. 

> Matching the first packet in a "flow" against DTLS packet format and blocking if it matches. If the first packet is anything else packets are permitted. UDP dst port does not seem to change this.